src/m365/teams/commands/channel/channel-remove.ts

```diff
@@ -55,7 +66,10 @@ class TeamsChannelRemoveCommand extends GraphCommand {
         option: '-n, --name [name]'
       },
       {
-        option: '-i, --teamId <teamId>'
+        option: '-i, --teamId [teamId]'
```

Comment:
As I wrote in the earlier comments for the MD files, could you do that here as well?